### ISSUE TITLE
Binary of cardano-cli version 9.0.0.1 not starting

### ISSUE THREAD

**Issue by KarstenSiebert** [input-output-hk/cardano-cli#831]
## Description

The comand line interface 9.0.0.1 does not start

## Steps to Reproduce

1. downloaded and extracted to server Ubuntu 22.04
2. chmod 755 (comes as 644)
3. ./cardano-cli-x86_64-linux --version
bash: ./cardano-cli-x86_64-linux: No such file or directory

## Possible Solution

4. Switched back to 9.0.0.0 which runs fine

**Comment by smelc** [comment]
@KarstenSiebert> can you give the output of `file ./cardano-cli-x86_64-linux`?

Also, when you say _downloaded_, can you be more specific? Did you obtain the binary from the [release pages](https://github.com/IntersectMBO/cardano-cli/releases)?

**Comment by KarstenSiebert** [comment]
The output is:

-bash: ./cardano-cli-x86_64-linux: No such file or directory

which I do not understand, as I had launched the program. I am attaching a shot of the command line interface, which shows the commands, I issued.

![issued-commands](https://github.com/user-attachments/assets/98e5cad2-abce-4725-9ee4-b0fbee18f8cf)

**Comment by KarstenSiebert** [comment]
The steps to reproduce also show, what I had done. Thanks.

**Comment by smelc** [comment]
Probably a duplicate of https://github.com/IntersectMBO/cardano-cli/issues/815

I can't observe a difference between `9.0.0.0` and `9.0.0.1`, and it works on my machine because the Nix store is populated.

`9.0.0.0`:

```shell
→ ./cardano-cli-x86_64-linux-9.0.0.0 --version
cardano-cli 9.0.0.0 - linux-x86_64 - ghc-9.6
git rev 0000000000000000000000000000000000000000
→ file ./cardano-cli-x86_64-linux-9.0.0.0
./cardano-cli-x86_64-linux-9.0.0.0: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /nix/store/aw2fw9ag10wr9pf0qk4nk5sxi0q0bn56-glibc-2.37-8/lib/ld-linux-x86-64.so.2, for GNU/Linux 3.10.0, not stripped
→ ldd ./cardano-cli-x86_64-linux-9.0.0.0
	linux-vdso.so.1 (0x00007ffd7d2d3000)
	libm.so.6 => /nix/store/aw2fw9ag10wr9pf0qk4nk5sxi0q0bn56-glibc-2.37-8/lib/libm.so.6 (0x00007f363379d000)
	libz.so.1 => /nix/store/4rx3vkkd91wkbhpflsplfga603cp1l1c-zlib-1.3/lib/libz.so.1 (0x00007f363377f000)
	libsodium.so.23 => /nix/store/srvp94k0f4kq70mgy94fgxml742lb30w-libsodium-vrf-dbb48cc/lib/libsodium.so.23 (0x00007f363371d000)
	libblst.so => /nix/store/pb1g3ayg0qljjfq0j340sbi2rpdr32gx-blst-03b5124/lib/libblst.so (0x00007f36336f0000)
	libsecp256k1.so.2 => /nix/store/7q573ggd63yaypmxbf6qicyshixv2j1r-secp256k1-acf5c55/lib/libsecp256k1.so.2 (0x00007f36335c9000)
	libpthread.so.0 => /nix/store/aw2fw9ag10wr9pf0qk4nk5sxi0q0bn56-glibc-2.37-8/lib/libpthread.so.0 (0x00007f36335c4000)
	libstdc++.so.6 => /nix/store/xpxln7rqi3pq4m0xpnawhxb2gs0mn1s0-gcc-12.3.0-lib/lib/libstdc++.so.6 (0x00007f3633200000)
	libgmp.so.10 => /nix/store/8zkv8ylir9ydi2y45j21s7nq1gmxvdd5-gmp-with-cxx-6.3.0/lib/libgmp.so.10 (0x00007f363351f000)
	libc.so.6 => /nix/store/aw2fw9ag10wr9pf0qk4nk5sxi0q0bn56-glibc-2.37-8/lib/libc.so.6 (0x00007f363301a000)
	librt.so.1 => /nix/store/aw2fw9ag10wr9pf0qk4nk5sxi0q0bn56-glibc-2.37-8/lib/librt.so.1 (0x00007f363351a000)
	libdl.so.2 => /nix/store/aw2fw9ag10wr9pf0qk4nk5sxi0q0bn56-glibc-2.37-8/lib/libdl.so.2 (0x00007f3633515000)
	libffi.so.8 => /nix/store/xaz82jmpn8xj9y0v5jxxlrqrwpll832p-libffi-3.4.4/lib/libffi.so.8 (0x00007f3633508000)
	libnuma.so.1 => /nix/store/v50h09mk3jpqdqcx7bdwg6c6zvjyyx5a-numactl-2.0.16/lib/libnuma.so.1 (0x00007f36334fa000)
	libgcc_s.so.1 => /nix/store/xpxln7rqi3pq4m0xpnawhxb2gs0mn1s0-gcc-12.3.0-lib/lib/libgcc_s.so.1 (0x00007f36334d7000)
	/nix/store/aw2fw9ag10wr9pf0qk4nk5sxi0q0bn56-glibc-2.37-8/lib/ld-linux-x86-64.so.2 => /lib64/ld-linux-x86-64.so.2 (0x00007f363387f000)
```

`9.0.0.1`:

```shell
→ ./cardano-cli-x86_64-linux-9.0.0.1 --version
cardano-cli 9.0.0.1 - linux-x86_64 - ghc-9.6
git rev 0000000000000000000000000000000000000000
→ file ./cardano-cli-x86_64-linux-9.0.0.1
./cardano-cli-x86_64-linux-9.0.0.1: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /nix/store/aw2fw9ag10wr9pf0qk4nk5sxi0q0bn56-glibc-2.37-8/lib/ld-linux-x86-64.so.2, for GNU/Linux 3.10.0, not stripped
→ ldd ./cardano-cli-x86_64-linux-9.0.0.1
	linux-vdso.so.1 (0x00007ffee4ded000)
	libm.so.6 => /nix/store/aw2fw9ag10wr9pf0qk4nk5sxi0q0bn56-glibc-2.37-8/lib/libm.so.6 (0x00007ef60f673000)
	libz.so.1 => /nix/store/4rx3vkkd91wkbhpflsplfga603cp1l1c-zlib-1.3/lib/libz.so.1 (0x00007ef60f655000)
	libsodium.so.23 => /nix/store/srvp94k0f4kq70mgy94fgxml742lb30w-libsodium-vrf-dbb48cc/lib/libsodium.so.23 (0x00007ef60f5f3000)
	libblst.so => /nix/store/pb1g3ayg0qljjfq0j340sbi2rpdr32gx-blst-03b5124/lib/libblst.so (0x00007ef60f5c6000)
	libsecp256k1.so.2 => /nix/store/7q573ggd63yaypmxbf6qicyshixv2j1r-secp256k1-acf5c55/lib/libsecp256k1.so.2 (0x00007ef60f49f000)
	libpthread.so.0 => /nix/store/aw2fw9ag10wr9pf0qk4nk5sxi0q0bn56-glibc-2.37-8/lib/libpthread.so.0 (0x00007ef60f49a000)
	libstdc++.so.6 => /nix/store/xpxln7rqi3pq4m0xpnawhxb2gs0mn1s0-gcc-12.3.0-lib/lib/libstdc++.so.6 (0x00007ef60f200000)
	libgmp.so.10 => /nix/store/8zkv8ylir9ydi2y45j21s7nq1gmxvdd5-gmp-with-cxx-6.3.0/lib/libgmp.so.10 (0x00007ef60f15d000)
	libc.so.6 => /nix/store/aw2fw9ag10wr9pf0qk4nk5sxi0q0bn56-glibc-2.37-8/lib/libc.so.6 (0x00007ef60ef77000)
	librt.so.1 => /nix/store/aw2fw9ag10wr9pf0qk4nk5sxi0q0bn56-glibc-2.37-8/lib/librt.so.1 (0x00007ef60f493000)
	libdl.so.2 => /nix/store/aw2fw9ag10wr9pf0qk4nk5sxi0q0bn56-glibc-2.37-8/lib/libdl.so.2 (0x00007ef60f48e000)
	libffi.so.8 => /nix/store/xaz82jmpn8xj9y0v5jxxlrqrwpll832p-libffi-3.4.4/lib/libffi.so.8 (0x00007ef60f481000)
	libnuma.so.1 => /nix/store/v50h09mk3jpqdqcx7bdwg6c6zvjyyx5a-numactl-2.0.16/lib/libnuma.so.1 (0x00007ef60f473000)
	libgcc_s.so.1 => /nix/store/xpxln7rqi3pq4m0xpnawhxb2gs0mn1s0-gcc-12.3.0-lib/lib/libgcc_s.so.1 (0x00007ef60f450000)
	/nix/store/aw2fw9ag10wr9pf0qk4nk5sxi0q0bn56-glibc-2.37-8/lib/ld-linux-x86-64.so.2 => /lib64/ld-linux-x86-64.so.2 (0x00007ef60f755000)
```

**Comment by KarstenSiebert** [comment]
Oh, I see, yes the same issue. I do not have nix installed, so the binary will not launch without the dyn libs. It would be usefull to publish a statically linked binary, if just the binary is published, like the entire 9.0.0 package. Thank you. I close this.

**Comment by smelc** [comment]
@KarstenSiebert> yep the fix is around the corner: https://github.com/IntersectMBO/cardano-cli/pull/819 We'll be publishing statically linked binaries very soon :+1:
